File: eform-client/src/app/plugins/modules/workorders-pn/models/work-orders-settings.model.ts
import {SiteNameDto} from 'src/app/common/models';

export class WorkOrdersSettingsModel {
  folderId?: number;
  folderTasksId?: number;
  folderName: string;
  folderTasksName: string;
  assignedSites: SiteNameDto[] = [];
}

File: eFormAPI/Plugins/WorkOrders.Pn/WorkOrders.Pn/Infrastructure/Models/Settings/WorkOrdersSettingsModel.cs
using System.Collections.Generic;

namespace WorkOrders.Pn.Infrastructure.Models.Settings
{
    public class WorkOrdersSettingsModel
    {
        public int? FolderId { get; set; }
        public int? FolderTasksId { get; set; }
        public List<SiteNameModel> AssignedSites { get; set; } = new List<SiteNameModel>();
    }
}
